packages/frontend/src/components/connect-button/account-menu/activities/activity/index.tsx

```diff
@@ -9,6 +9,7 @@ import classNames from "classnames";
 import { formatTokenAmount } from "@/src/utils/format";
 import { trackFathomEvent } from "@/src/utils/fathom";
 import { RemoteLogo } from "@/src/components/remote-logo";
+import { formatUnits } from "viem";
```

Comment:
Why did we import `formatUnits`?

Reply:
Fixed in b735f26b5dfbc13b3a57dba3b75733a8427e98ca.